lib/object/games_obj.dart

```diff
@@ -2,26 +2,41 @@ class GameContent {
   String title;
   String description;
   String icon;
+  String score;
+  String percentAchieve;
+  String winnerCup;
 
   GameContent(
       {this.title = 'Need for Speed Payback',
       this.description = 'Jump back in',
-      this.icon = 'assets/images/img_1.png'});
+      this.icon = 'assets/images/img_1.png',
```

Comment:
I like the consistency of image naming. The only improvement I'd suggest is making those name more informative, so you can figure out what this image is by it's name, for example: `assets/images/game_1_nfs.png` and so on